cinema/migrations/0003_movie_duration.py

```diff
@@ -6,13 +6,13 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('cinema', '0002_initial'),
+        ("cinema", "0002_initial"),
     ]
 
     operations = [
         migrations.AddField(
-            model_name='movie',
-            name='duration',
+            model_name="movie",
+            name="duration",
             field=models.IntegerField(default=123),
             preserve_default=False,
```

Comment:
The `default` value is set to 123, but `preserve_default` is set to `False`. This means that the default value will not be preserved in the database schema. If you intend to have a default value of 123 for existing records, consider setting `preserve_default` to `True` or handling the default value assignment in a data migration.